bin/mini_tree_server.rs

```diff
@@ -1,96 +1,19 @@
-use axum::{extract::Json, http::StatusCode, response::IntoResponse, routing::post, Router};
-use const_env::from_env;
-use num_bigint::BigUint;
+use axum::{routing::post, Router};
 use std::net::SocketAddr;
-use summa_backend::merkle_sum_tree::{Entry, MerkleSumTree, Node, Tree};
 
-use serde::{Deserialize, Serialize};
-
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct JsonNode {
-    pub hash: String,
-    pub balances: Vec<String>,
-}
-
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct JsonEntry {
-    balances: Vec<String>,
-    username: String,
-}
-
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct JsonMerkleSumTree {
-    root: JsonNode,
-    nodes: Vec<Vec<JsonNode>>,
-    depth: usize,
-    entries: Vec<JsonEntry>,
-    is_sorted: bool,
-}
-
-#[from_env]
-const N_ASSETS: usize = 2;
-#[from_env]
-const N_BYTES: usize = 14;
+use summa_aggregation::mini_tree_generator::create_mst;
 
 #[tokio::main]
 async fn main() {
     // Define the app with a route
     let app = Router::new().route("/", post(create_mst));
 
     // Define the address to serve on
-    let addr = SocketAddr::from(([0, 0, 0, 0], 4000)); // TODO: assign ports from env variable
+    let addr = SocketAddr::from(([0, 0, 0, 0], 4000));
 
     // Start the server
     axum::Server::bind(&addr)
```

Comment:
I see that there are methods such as `convert_node_to_json` and `create_mst` that already live inside the `executor`. Is there a way to avoid duplicates?

Reply:
I will refactor these helper functions before merging.
I think there is no duplicates, but it's not well organized.